src/convertToObject.js

```diff
@@ -13,6 +13,24 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const sourceArray = sourceString.split(';');
+  const property = [];
```

Comment:
this is an array of properties, right?

```suggestion
  const properties = [];
```